js/north.js

```diff
@@ -379,6 +379,7 @@ jQuery( function ( $ ) {
 	// Handle smooth scrolling
 	if ( siteoriginNorth.smoothScroll ) {
 		$( '#site-navigation a[href*="#"]:not([href="#"])' ).northSmoothScroll();
+		$( 'a[href*="#"]:not([href="#"])' ).northSmoothScroll();
```

Comment:
It would be a little neater to join these, since they're calling the same `northSmoothScroll` jQuery function. So either: `$( '#site-navigation a[href*="#"]:not([href="#"]), a[href*="#"]:not([href="#"])' ).northSmoothScroll();` or `$( '#site-navigation a[href*="#"]:not([href="#"])' ).add( 'a[href*="#"]:not([href="#"])' ).northSmoothScroll();`. Haven't tested, but either should work.